gstcefsrc.h

```diff
@@ -69,7 +69,8 @@ class App : public CefApp, public CefBrowserProcessHandler {
   void OnScheduleMessagePumpWork(int64_t delay_ms) override;
 #endif
 private:
-  IMPLEMENT_REFCOUNTING(App);
+  IMPLEMENT_REFCOUNTING(BrowserApp);
+  DISALLOW_COPY_AND_ASSIGN(BrowserApp);
```

Comment:
I'm curious what that is for?

Reply:
To be honest my thinking didn't go beyond "the sample has it, so maybe I should add it" - https://github.com/chromiumembedded/cef-project/blob/aaccb78661c390a5ce6650df8db54ea5424d431c/examples/message_router/app_browser_impl.cc#L53

Reply:
OK, let's not add it here then I think :)